src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+
+import styles from './Slider.module.scss';
+import classNames from 'classnames/bind';
+const cx = classNames.bind(styles);
+
+import { Root as Root, Track as Track, Range as Range, Thumb as Thumb } from '@radix-ui/react-slider';
+
+export interface SliderProps {
+  step?: number;
+  min?: number;
+  max?: number;
+  value?: number;
+  className?: string;
+  minStep?: number;
+  isLight?: boolean;
+  isSmall?: boolean;
+  isRangeSlider?: boolean;
+  label?: string;
+  onValueChange?: (value: number[]) => void;
+}
+
+export const Slider: React.FC<SliderProps> = ({
+  step,
+  min,
+  max,
+  value,
+  isSmall,
+  isLight,
+  isRangeSlider,
```

Comment:
Is `isRangeSlider` necessary here? It looks like a range slider requires `min` and `max` props to be defined, so `isRangeSlider` seems functionality equivalent to `has min and max`. I could be missing something here, but looks like we `isRangeSlider === Boolean(min) && Boolean(max)`, in which case we should remove it as it makes the slider a bit more complicated to configure. If there's something I'm missing let me know, there might be some edge cases I'm not considering.

If you wanted to do some Typescript trickery, you could do something like
```ts
interface SliderProps {
    // all values apart from min and max
}

interface RangeSliderProps extends SliderProps {
    min: number;
    max: number;
}

export const Slider = ({ ... }: SliderProps | RangeSliderProps) => ...
```

This would ensure that a standard Slider wouldn't have `min` and `max`, and a range slider will always need both `min` and `max`. I can't remember the exact Typescript syntax so might be slightly off!

Reply:
Both sliders will need a min and max as they dictate the range that the slider functions within. The only difference is with the rangeSlider, it uses the min and max as default values as well.

Reply:
I see - makes sense! All good as is then 😊